content/altinn-studio/guides/monitor-and-instrument/_index.nb.md

```diff
@@ -0,0 +1,121 @@
+
+Det nye monitorerings- og instrumenteringsoppsettet basert på OpenTelemetry kan aktiveres ved å sette 
+`UseOpenTelemetry` til `true` i `appsettings.json` eller ekvivalent.
+
+{{< highlight json "linenos=false,hl_lines=3" >}}
+{
+  "AppSettings": {
+    "UseOpenTelemetry": true
+  }
+}
+{{< / highlight >}}
+
+Når appen kjøres med denne instillingen så vil Altinn.App biblioteket sende telemetri til localtest ved lokal kjøring
+og til Azure Monitor ved kjøring i et miljø.
+
+[Les mer om konfigurasjonsmuligheter på referansesiden for konfigurasjon](/nb/altinn-studio/reference/monitoring/configuration).
+
+## Egendefinert instrumentering
+
+Vi illustrerer egendefinert instrumentering med et eksempel. I `Program.cs` legger vi til en simpel `IHostedService` implementasjon
+som kan instrumenteres til å eksponere telemetri.
+
+Telemetri- og instrumentering-API'ene i Altinn.App biblioteket bli eksponert gjennom `Telemetry`-klassen. Det er
+et trådsikkert singleton-objekt tilgjengelig i dependency injection containeren.
+La oss utvide `Program.cs` til å inkludere dette.
+
+Hvis de ikke allerede er der, så trenger vi de følgende avhengighetene på toppen av filen:
+
+```csharp
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Hosting;
+using Altinn.App.Core.Features;
+```
+
+Så kan vi implementere den følgende klassen nederst i filen:
+
+```csharp
+sealed class StartupService(ILogger<StartupService> logger, Telemetry telemetry) : IHostedService
+{
+    public async Task StartAsync(CancellationToken cancellationToken)
+    {
+        // Lag en egendefinert metrikk - en teller i dette tilfellet. Navnet blir `altinn_app_started`
+        var counter = telemetry.Meter.CreateCounter<long>(Telemetry.Metrics.CreateName("started"));
+        // Inkrementer telleren
+        counter.Add(1);
+
+        // Start en aktivitet, som blir utsendt som et OTel span
+        using var activity = telemetry.ActivitySource.StartActivity("StartupService");
+        {
+            // Vent litt, deretter legg til en underaktivitet
+            await Task.Delay(100, cancellationToken);
+            using var childActivity = telemetry.ActivitySource.StartActivity("ChildActivity");
+            await Task.Delay(100, cancellationToken);
+        }
+
+        // Logger kommer fra `ILogger<T>` interfacet
+        var now = DateTimeOffset.UtcNow;
+        logger.LogInformation("StartupService logging - Now={Now}", now);
+    }
+
+    public Task StopAsync(CancellationToken cancellationToken) => Task.CompletedTask;
+}
+```
+
+[Les mer om instrumenteringsmuligheter på referansesiden for instrumentering](/nb/altinn-studio/reference/monitoring/instrumentation).
+
+## Visualisering
+
+Her er en kort oversikt over visualisering av telemetrien vi instrumenterte over.
+
+[Les mer om visualisering av telemetri på referansesiden for visualisering](/nb/altinn-studio/reference/monitoring/visualisation).
```

Comment:
Når man kommer til denne siden har man ikke fått noe informasjon om visualisering enda. Dermed vet man jo ikke om man trenger å navigere videre for å lese mer, fordi man ikke vet hvilken informasjon som finnes på denne siden enda.